templates/repo/code/recently_pushed_new_branches.tmpl

```diff
@@ -0,0 +1,11 @@
+{{range .RecentlyPushedNewBranches}}
+	<div class="ui positive message gt-p-4 gt-df gt-ac gt-tl">
+		<div class="gt-f1">
+			{{$timeSince := TimeSince .UpdatedUnix.AsTime $.locale}}
+			{{$.locale.Tr "repo.pulls.recently_pushed_new_branches" $.Repository.OwnerName $.Repository.LowerName .Name $timeSince | Safe}}
```

Comment:
I guess the `.Name` can't be `Safe` because it comes from git repo.

Reply:
fixed.